ci-operator/step-registry/telco5g/sno-cnf/telco5g-sno-cnf-workflow.yaml

```diff
@@ -5,6 +5,7 @@ workflow:
     - ref: telco5g-jobs-router
     - ref: telco5g-sno-setup
     test:
+    - ref: telco5g-sno-cnf-tests
     - ref: telco5g-cnf-tests
```

Comment:
https://github.com/openshift/release/pull/40003#issuecomment-1587085270

Reply:
@stbenjam Then how to solve this? combine all tests in one ref?